examples/nodejs_control_interface/src/main.js

```diff
@@ -25,7 +25,7 @@ function create_request_to_add_new_workload(root) {
                             dynamic_nginx: {
                                 agent: "agent_A",
                                 runtime: "podman",
-                                restart: true,
+                                restart: NEVER,
```

Comment:
FYI: examples in other languages have this flag too. Check them too, please.

Reply:
I will test the examples tomorrow. I get the known podman DNS lookup error. I have to fix this first.

Reply:
The examples are adapted now.